src/runtime_src/core/tools/xbtracer/src/app/launcher.cpp

```diff
@@ -175,15 +255,82 @@ int parse_cmdline(launcher& app, int argc, char* argv[])
   if (optind == argc)
     log_f("There should be alleast 1 argument without option switch");
 
-  split_by_ws(argv[argc-1], app.m_child_cmd_args);
-  for (auto& element : app.m_child_cmd_args)
+  if (optind <= static_cast<int>(args.size() - 1))
```

Comment:
Shouldn't the cast be on the other size?  `static_cast<size_t>(optind)`  smaller to larger?

Reply:
Correct, fixed in update commit